package.json

```diff
@@ -14,7 +14,7 @@
     "build": "nx run core:build",
     "build:server": "http-server dist",
     "dev": "nx run core:dev",
-    "dev:server": "http-server build",
+    "dev:server": "http-server build --cors",
```

Comment:
What does this do?

Reply:
Possibly after [this](https://github.com/bigcommerce/checkout-js/pull/2020) update, when we try and run `dev:server` it turns runs with `CORS: false` which does not allow dev store to make requests to localhost, so adding this flag makes that http server setting `CORS: true` true which allows dev store to run fine